configs/config.dell_optiplex_9010_sff_uefi_txt

```diff
@@ -0,0 +1,53 @@
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_USE_BLOBS=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+# CONFIG_DRIVERS_UART_8250IO is not set
+CONFIG_PCIEXP_CLK_PM=y
+CONFIG_HAVE_EM100_SUPPORT=y
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/evaluation_logo.bmp"
```

Comment:
I would switch to normal logo. Or it was supposed to be evaluation?
```suggestion
CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
```

Reply:
Fixed in [this commit](https://github.com/Dasharo/coreboot/pull/541/commits/a4143cefd7ede989e03dd623ba3e1e8c288f2775)